p2-maven-plugin/src/main/java/org/eclipse/tycho/p2maven/repository/P2RepositoryManager.java

```diff
@@ -47,6 +47,8 @@
 @Component(role = P2RepositoryManager.class)
 public class P2RepositoryManager {
 
+	private static final int RETRY_ATTEMPTS = 3;
```

Comment:
I'm not overly happy with this magic number. Perhaps there's a better way to define the number of retries? Perhaps it should even be configurable?

Reply:
You can define a property and then use `MavenPropertyHelper` (don't forget to document the property [here](https://github.com/eclipse-tycho/tycho/blob/main/src/site/markdown/SystemProperties.md))

Reply:
Done